tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -315,14 +315,26 @@ def test_positive_install_sat_with_katello_certs(self, vm_setup):
         :CaseAutomation: Automated
         """
         cert_data, rhel_vm = vm_setup
+        version = str(settings.server.version.rhel_version)[0]
```

Comment:
Again I don't think I would read from settings here, I would reference `self.satellite.os_version`, which will give you back a Version instance that you can then reference the `major` attribute here.